frontends/web/src/api/exchanges.ts

```diff
@@ -99,6 +99,28 @@ export const getPocketURL = (action: TExchangeAction): Promise<TPocketUrlRespons
   return apiGet(`exchange/pocket/api-url/${action}`);
 };
 
+export type TBTCDirectInfoResponse = {
+  success: true;
+  url: string;
+  apiKey: string;
+} | {
+  success: false;
+  errorMessage: string;
+};
+
+export const getBTCDirectInfo = (
+  action: TExchangeAction,
+): Promise<TBTCDirectInfoResponse> => {
+  console.log(action);
+  // TODO: change to return apiGet(`exchange/btc-direct/info/${action}`); or similar
+  return Promise.resolve({
+    success: true,
+    url: '/btcdirect/fiat-to-coin.html', // local static file for testing
```

Comment:
I expect that in production mode we will point to a subdomain where we host the content on fiat-to-coin.html